src/VirtualMonoRepo/eng/common/templates/jobs/vmr-synchronization.yml

```diff
@@ -0,0 +1,76 @@
+###
+### DO NOT MODIFY THIS FILE!
+###
+### This YAML was auto-generated from VmrSyncJobTemplate
+### To make changes, change the C# definition and rebuild its project
+###
+### These steps synchronize code from product repositories into the VMR (https://github.com/dotnet/dotnet)
+
+parameters:
+- name: targetRef
+  displayName: Target revision in dotnet/installer to synchronize
+  type: string
+  default: $(Build.SourceVersion)
+
+- name: vmrBranch
+  displayName: dotnet/dotnet branch to use
+  type: string
+  default: $(Build.SourceBranchName)
+
+jobs:
+- job: Synchronize
+  displayName: Synchronize dotnet/dotnet
+  pool:
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      name: NetCore-Public
+      demands:
+      - ImageOverride -equals Build.Ubuntu.2004.Amd64.Open
+    ${{ else }}:
+      name: NetCore1ESPool-Internal
+      demands:
+      - ImageOverride -equals Build.Ubuntu.2004.Amd64
+  steps:
+  - checkout: self
+    clean: true
+    displayName: Checkout dotnet/installer
+    fetchDepth: 0
+    path: installer
+
+  - checkout: vmr
+    clean: true
+    displayName: Checkout dotnet/dotnet
+    fetchDepth: 0
+    path: vmr
+    timeoutInMinutes: 30
+
+  - script: |-
+      source ./eng/common/tools.sh
+      InitializeDotNetCli true
+      ./.dotnet/dotnet tool restore
+    displayName: Restore toolset
+    workingDirectory: $(Agent.BuildDirectory)/installer
+
+  - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+    - script: |-
+        cp -r $(Agent.BuildDirectory)/installer $(Agent.TempDirectory)/installer
+      displayName: '[PR Only] Prepare dotnet/installer clone'
+
+  - script: |-
+      $(Agent.BuildDirectory)/installer/.dotnet/dotnet darc vmr update --vmr $(Agent.BuildDirectory)/vmr --tmp $(Agent.TempDirectory) --azdev-pat $(System.AccessToken) --github-pat $(BotAccount-dotnet-bot-repo-PAT) --recursive --verbose installer:${{ parameters.targetRef }}
+    displayName: Synchronize dotnet/dotnet
+    workingDirectory: $(Agent.BuildDirectory)/installer
+    timeoutInMinutes: 90
```

Comment:
Wow, it takes this long?

Reply:
Replied in https://github.com/dotnet/installer/pull/14805#discussion_r1003232203